recipe/build.sh

```diff
@@ -131,7 +131,7 @@ if [ $(uname) == Darwin ]; then
                 -system-sqlite \
                 -no-framework \
                 -arch $(uname -m) \
-                -platform unsupported/macx-clang-libc++ \
```

Comment:
Why are you mixing Clang and g++ stuff? Are you sure this is the right way of proceeding?

Reply:
My mistake! 
![image](https://user-images.githubusercontent.com/7598233/54087615-d6b4d580-4354-11e9-8436-4bce46ca45ab.png)
I think we're using only clang now. [Could you take a look](https://github.com/conda-forge/qt-feedstock/blob/1e3c365108cca781f1f594ef847ba0213f7c532d/recipe/build.sh)? 

Still we're having some sneaky errors. I was about to try to assign `c++11` to the CONFIG on the `qmake.conf`. Or maybe we will need to patch that thingy

Reply:
The latest error was something like `Structure.h:320:117: error: non-const reference cannot bind to bit-field 'm_attributesInPrevious'`